### PR TITLE
Features goes to 6.x branch

### PR DESCRIPTION
**Pull request by jongotlin**
| Q             | A
| ------------- | ---
| Branch?       | 6.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by fancyweb** [comment]
Sorry, can you rebase and change the target to 4.4 please?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Additionally (see https://symfony.com/releases):
  - Always add tests and ensure they pass.
  - Bug fixes must be submitted against the lowest maintained branch where they apply
    (lowest branches are regularly merged to upper ones so they get the fixes too.)
- - Features and deprecations must be submitted against branch 5.x.
+ - Features and deprecations must be submitted against branch 6.x.
```
Comment (fancyweb) [inline .github/PULL_REQUEST_TEMPLATE.md]: 6.1?

Comment (fancyweb) [reply .github/PULL_REQUEST_TEMPLATE.md]: I don't know if we need a specific branch here actually, it's already above :man_shrugging:

Comment (jongotlin) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes, it's kinda redundant. Maybe we should remove the whole line since it's stated above (except deprecations).

Comment (fabpot) [reply .github/PULL_REQUEST_TEMPLATE.md]: Maybe:

```suggestion
 - Features and deprecations must be submitted against the latest branch.
```

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @jongotlin.

**Pull request closed without merging**